.changeset/perfect-ducks-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+"xstate-viz-app": patch
+---
+
+Added a welcome area for new users to help new folks understand what the viz can do.
```

Comment:
A screenshot of this new "screen" would look great in the release notes :P

Reply:
How, sensei?

Reply:
1. upload to any location - u can even upload to GitHub by dropping a screenshot into the comment box and once the upload is done just copy-paste the result
2. use `<img/>` tag in the changeset file 😉